rackspace.yaml

```diff
@@ -4,6 +4,7 @@ application-name: Drupal
 application-version: 7.32
 flavor: Multiple Linux servers
 flavor-weight: 1
+popularity: 5
```

Comment:
Drupal-multi was disabled by Jason Boyles a while back as he was doing a rework on it. I imagine this ranking might have come from already created stacks.